assets/js/main.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+function generateColors(searchParams) {
+  var primary = 360 * Math.random();
+  var secondary = 360 * Math.random();
+  var lightness = 100 * Math.random();
```

Comment:
maybe we can round these numbers. for example:

```suggestion
  var lightness = Math.round(100 * Math.random());
```

This will make the URLs look a bit nicer!